src/main/java/org/openRealmOfStars/starMap/planet/Planet.java

```diff
@@ -964,19 +976,22 @@ private int getTotalCreditProduction() {
       sb.append(getTax());
       sb.append("<br>");
     }
-    if (eventFound && event.getExtraCredit() != 0) {
-      value = event.getExtraCredit();
+    int statusesBonus = statuses.stream()
+        .map(status -> status.getStatus().getCredBonus())
+        .reduce(0, Integer::sum);
+    if (statusesBonus != 0) {
+      value = statusesBonus;
       result = result + value;
       if (value > 0) {
         sb.append("<li> ");
-        sb.append(event.getName());
+        sb.append("Planetary Statuses");
```

Comment:
Should this be name of the planetary status?

Reply:
What do you mean by description generation consume absurd amount of computing power? Currently those are just count couple of numbers, using string builder to textual info about it. That should not be CPU expensive.

Reply:
At the moment longest time spend in AI Turn is while drawing the screen with text " Please wait...". I have made there a delay so it will at least stay on screen 6*75ms=450ms.

But true that getGuideSuggestion() could store those results locally, it even calls research and credit production twice. Which kind of proofs that those calculations aren't CPU expensive. Memory usage is also quite stable. It used to be around 1.2GB of RAM, but recently it has increased to 1.7GB of RAM.

Reason why info text collection and actual calculation are in same method is that it is easier to update one and then update the other side too. If these would be separate method, it is very like at some point these can be different.